src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -144,6 +144,11 @@ public function isFillable($key)
             return true;
         }
 
+        // If there's a method, chances are it's a relationship.
+        if (method_exists($this, $key)) {
+            return true;
```

Comment:
This implicit assumption looks like a possible attack vector to me #SecurityFirst

Reply:
@mfn ah yes, now that you mention it, it will become a security risk; better to leave it for the dev to decide if they want to make it fillable or not.